resolwe_bio/processes/support_processors/multiqc.py

```diff
@@ -437,7 +437,7 @@ class MultiQC(Process):
     }
     category = "QC"
     data_name = "MultiQC report"
-    version = "1.25.0"
+    version = "1.26.0"
```

Comment:
Docker image version will need to be bumped to use the updated MultiQC config file.